core/src/main/java/com/sequenceiq/cloudbreak/rotation/context/RedbeamsPollerRotationContext.java

```diff
@@ -0,0 +1,18 @@
+package com.sequenceiq.cloudbreak.rotation.context;
+
+import com.sequenceiq.cloudbreak.rotation.secret.RotationContext;
+import com.sequenceiq.cloudbreak.rotation.secret.type.RedbeamsSecretType;
+
+public class RedbeamsPollerRotationContext extends RotationContext {
```

Comment:
would it makes sense to create only one poller context per service in common? I guess all of them will have the same structure